packages/manager/modules/iam/src/iam.constants.js

```diff
@@ -31,49 +31,111 @@ const ENTITY = {
 
 const ENTITY_NAME_PATTERN = /^[a-zA-Z0-9-/_+]*$/;
 
-// TODO wait for other guide specifications
 const GUIDE = {
   IAM: {
-    FR: 'https://docs.ovh.com/fr/customer/IAM/',
+    DEFAULT:
+      'https://help.ovhcloud.com/csm/en-ie-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056806',
+    ASIA:
+      'https://help.ovhcloud.com/csm/asia-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056797',
+    AU:
+      'https://help.ovhcloud.com/csm/en-au-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056803',
+    CA:
+      'https://help.ovhcloud.com/csm/en-ca-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056802',
+    DE:
+      'https://help.ovhcloud.com/csm/de-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056810',
+    ES:
+      'https://help.ovhcloud.com/csm/es-es-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056800',
+    FR:
+      'https://help.ovhcloud.com/csm/fr-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056808',
+    GB:
+      'https://help.ovhcloud.com/csm/en-gb-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056801',
+    IT:
+      'https://help.ovhcloud.com/csm/it-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056796',
+    PL:
+      'https://help.ovhcloud.com/csm/pl-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056807',
+    PT:
+      'https://help.ovhcloud.com/csm/pt-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056798',
+    QC:
+      'https://help.ovhcloud.com/csm/fr-ca-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056804',
+    SG:
+      'https://help.ovhcloud.com/csm/en-sg-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056799',
+    US:
+      'https://help.ovhcloud.com/csm/en-ie-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056806',
+    WE:
+      'https://help.ovhcloud.com/csm/en-ie-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056806',
```

Comment:
Are we sure because normaly WE = en only note en-ie

Reply:
I agree but it is `en-ie` as you can see
![image](https://github.com/ovh/manager/assets/3984480/cddddb31-2623-47ae-8036-97b784eea6ba)